packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -408,11 +409,46 @@ export interface SdkModelPropertyTypeBase extends DecoratedType {
   crossLanguageDefinitionId: string;
 }
 
+export interface SerializationOptions {
```

Comment:
This needs some doc comments. Can multiple be set? None be set and if so, what does that mean?